### ISSUE TITLE
Unsubcribe button (rather than link) - how to add url?

### ISSUE THREAD

**Issue by gingerling** [mautic/mautic#4421]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | 
| Feature request? | Y?
| Enhancement? | 

## Description:

TL:DR - **Is putting {unsubscribe_url} in the url box of a button, the correct way to create an unsubscribe button?** If so, can this be added to the docs? I searched for 45 mins and couldn't find an answer to this.

![screenshot from 2017-07-14 14-22-26](https://user-images.githubusercontent.com/2853040/28214682-2b68749e-68a3-11e7-9cab-0f07ef44d10d.png)

List cleaning is about to become even more important: it is expected that there will, over the next few years, be more focus on the domain name / sending address and less on the sending IP.  The damage from dirty lists becomes a lot more scary. Therefore I think mautic could do more in terms of list cleaning workflow.
One common process is sending a "opt out" email or "opt back in" email to unresponsive subscribers. 

In this type of email an unsubscribe button is more appropriate than an unsubscribe link, however, it's not clear to me from reading docs, config section, searching google and the youtube docs, how to add the unsubscribe url to a button. 

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 
| PHP version | 

### Steps to reproduce:
1. 
2. 
 
### Log errors: 

_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._

**Comment by escopecz** [comment]
The answer is yes!

Please help us with the docs any time https://github.com/mautic/documentation#how-to-contribute-to-the-docs
